### PR TITLE
global: initial commit

### PR DESCRIPTION
**Pull request by nharraud**
- Adds all required files for a standard project.

Signed-off-by: Nicolas Harraudeau nicolas.harraudeau@cern.ch

### CODE REVIEW

**Comment by nharraud** [comment]
@tiborsimko I don't seem to have the permissions to assign this PR for review.

**Comment by tiborsimko** [comment]
@nharraud The rights should be OK now.

**Comment by nharraud** [comment]
@jirikuncar This is just adding all the standard files and test skeleton. I'll create an Issue describing the specs of the first function after that.

Created with cookiecutter, Babel and Flask removed.

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+Authors
+=======
+
+JSON Schema utility functions and commands.
+
+- CERN <info@invenio-software.org>
```
Comment (jirikuncar) [inline AUTHORS.rst:30]: @nharraud it should be you ;)

Comment (nharraud) [inline AUTHORS.rst:30]: Ah yeah you are right. Thanks for the reminder. It will be fixed in a later commit.

**Pull request merged**